rs/canister_sandbox/src/replica_controller/sandboxed_execution_controller.rs

```diff
@@ -286,7 +286,7 @@ impl SandboxedExecutionMetrics {
             sandboxed_execution_executed_message_slices: metrics_registry.int_counter_vec(
                 "sandboxed_execution_executed_message_slices_total",
                 "Number of executed message slices by type and status.",
-                &["api_type", "status"],
+                &["api_type", "status", "wasm_execution_mode"],
             ),
```

Comment:
There are [quite a few uses](https://github.com/search?q=repo%3Adfinity-ops%2Fk8s+sandboxed_execution_executed_message_slices_total+path%3A%2F%5Ebases%5C%2Fapps%5C%2F%2F&type=code) of `sandboxed_execution_executed_message_slices_total` in alerts and dashboards. Please update those (where necessary) before merging this change.

Reply:
Thanks @alin-at-dfinity. I have actually thought about the dashboards, but there we would likely get 2 entries, one for each execution mode, which is probably ok. I did indeed miss the alerts, I assume what we'd want to do there is just match on both wasm32/wasm64?